src/kvirc/ui/KviMenuBar.cpp

```diff
@@ -174,6 +176,14 @@ void KviMenuBar::setupSettingsPopup(QMenu *pop)
 	QAction *pAction = opt->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Toolbar)),__tr2qs("Toolbars"));
 	pAction->setMenu(m_pToolbarsPopup);
 
+#ifndef COMPILE_ON_MAC
+	m_pMenuBarAction = opt->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::StatusBar)),__tr2qs("Show Menu Bar"),m_pFrm,SLOT(toggleMenuBar()));
```

Comment:
This option doesnt show anything, it hides it, so logic fail.

Why also would the option name affect checkbox selection? what?